### PR TITLE
Use fa icons in calendar's day view

### PR DESCRIPTION
**Pull request by mdsupport**

### CODE REVIEW

**Comment by robertdown** [comment]
I have similar code ready for a PR. We should compare before we pull in

On Sep 24, 2016, at 16:11, mdsupport <notifications@github.com<mailto:notifications@github.com>> wrote:

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/openemr/openemr/pull/275

Commit Summary

  *   Use fa icons in calendar's day view

File Changes

  *   M interface/main/calendar/modules/PostCalendar/pntemplates/default/views/day/ajax_template.html<https://github.com/openemr/openemr/pull/275/files#diff-0> (27)
  *   M interface/main/calendar/modules/PostCalendar/pntemplates/default/views/header.html<https://github.com/openemr/openemr/pull/275/files#diff-1> (2)

Patch Links:

  *   https://github.com/openemr/openemr/pull/275.patch
  *   https://github.com/openemr/openemr/pull/275.diff

-

**Comment by mdsupport** [comment]
This deals only with the day view that we use.  From design perspective our users like icons for actions and text for medical information.  But community may have other preference.  

This PR can be closed anytime.

**Comment by robertdown** [comment]
Just looked at the code for this commit, my PR will encompass all these changes. It's already done, will submit this weekend. Also makes some other visual changes to the calendar.

@bradymiller Why are we not utilizing a global navigation and sideview on the calendar regardless of the view being day/week/month? Can I make that happen?

File: interface/main/calendar/modules/PostCalendar/pntemplates/default/views/day/ajax_template.html [hunk 1]
```diff
@@ -154,30 +154,37 @@
 <input type="hidden" name="jumpdate" id="jumpdate" value="">
 <input type="hidden" name="viewtype" id="viewtype" value="[-php-]echo $viewtype;[-/php-]">
 [-php-]
-echo "   <a href='#' value='" .xl ("Add"). "' onclick='newEvt(1, 9, 00, $Date, 0, 0)' class='css_button'/><span>".xl("Add")."</span></a>\n";
-echo "   <a href='#' value='" . xl ("Search") .
-    "' onclick='top.restoreSession();location=\"index.php?module=PostCalendar&func=search\"' class='css_button'/><span>".xl("Search")."</span></a>\n";
+echo "   <label title='" .xl ("New appointment"). "' onclick='newEvt(1, 9, 00, $Date, 0, 0)' class='css_button'>
+            <i class='fa fa-plus' aria-hidden='true'></i></label>\n";
```
Comment (bradymiller) [inline interface/main/calendar/modules/PostCalendar/pntemplates/default/views/day/ajax_template.html]: @mdsupport ,
What does the aria-hidden do?

**Comment by bradymiller** [comment]
This is really nice. I am curious what the aria-hidden does. @robertdown and @mdsupport , I plan to extend this to the week and month views for now(will be very minimal work). Then can always incorporate Robert's more extensive changes down the road when they are ready.

**Comment by bradymiller** [comment]
Extended this to the week and month view here (also changed label links to a links since the hovering showed pointer for the label links):
https://github.com/openemr/openemr/pull/310
(also looks nice in the legacy themes now that they use Ray's css buttons)

**Comment by bradymiller** [comment]
Brought the following PR into the codebase:
https://github.com/openemr/openemr/pull/310

Will close this PR (can open another PR when bring in Robert's changes down the road)

**Pull request closed without merging**